lib/ButtonNew/ButtonPrimary.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cx from 'classnames';
+import { ButtonBase } from './ButtonBase';
+import { propTypes, defaultProps, sizes } from './common';
+
+function ButtonPrimary({
+  children,
+  className,
+  disabled,
+  connectedRight,
+  danger,
+  ...rest
+}) {
+  const classes = cx('text-white ', className, {
+    'bg-blue-primary': !danger,
+    'bg-red-primary': danger,
+    'bg-neutral-90 text-neutral-primary': disabled,
```

Comment:
💚You'll have to add some more tailwind config but you can use tailwind for disabled styles like so https://tailwindcss.com/docs/pseudo-class-variants/#disabled-v1-1-0 up to you wether you go for this since having them here is fine!

Reply:
I'm going to leave this as is for now, as I like the control we get with keeping this in JS rather than CSS.

I also believe (but haven't validated) that if we used CSS pseudo selector `disabled`, we would have to add the styles at the `ButtonBase` level, as `ButtonPrimary` etc render this component, which in turn renders the button. If so, I think this could cause the components to get a bit messier.